packages/dom/test/serialize-css.test.js

```diff
@@ -158,5 +158,58 @@ describe('serializeCSSOM', () => {
       shadowEl.shadowRoot.adoptedStyleSheets = [];
       shadowElChild.shadowRoot.adoptedStyleSheets = [];
     });
+
+    it('captures blob styleSheets', () => {
+      if (platform !== 'plain') {
+        return;
+      }
+      withExample('<div>BlobStyle</div>', { withShadow: false });
+
+      function generateBlobUrl(cssStyle) {
+        const blob = new window.Blob([cssStyle], { type: 'text/css' });
+        return window.URL.createObjectURL(blob);
+      }
+
+      function createLinkElement(blobURL) {
```

Comment:
Lets rename it `createStyleLinkElement`, `createLinkElement` suggests it might be a hyperlink